src/component/Person.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import PropTypes from 'prop-types'
+
+const Person = ({ name, age = null, sex, isMarried = null, partnerName = null }) => (
+  <>
+    <section className="Person">
+      <h2 className="Person__name">My name is {name}</h2>
```

Comment:
```suggestion
      <h2 className="Person__name">
        My name is {name}
      </h2>
```
For better readability better to move the content to the newlines